FlashSpace/Features/Workspaces/Models/MacApp.swift

```diff
@@ -55,16 +87,17 @@ struct MacApp: Codable, Hashable, Equatable {
             // V2
             let container = try decoder.container(keyedBy: CodingKeys.self)
             self.name = try container.decode(String.self, forKey: .name)
-            self.bundleIdentifier = try container.decode(String.self, forKey: .bundleIdentifier)
+            // TODO: THIS IS BROKEN - preserve existing encoding of bundle ID
+            self.identifier = try container.decode(MacAppIdentifier.self, forKey: .identifier)
```

Comment:
It doesn't work, because now there is a different key `identifier` instead of `bundleIdentifier` which is not backward-compatible and doesn't exist yet in the config.